src/dev/src/app/iconography/icon-selection.ts

```diff
@@ -4,15 +4,15 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component } from '@angular/core';
-import { ChartShapes } from '@clr/icons/shapes/chart';
-import { CommerceShapes } from '@clr/icons/shapes/commerce';
-import { CoreShapes } from '@clr/icons/shapes/core';
-import { EssentialShapes } from '@clr/icons/shapes/essential';
-import { MediaShapes } from '@clr/icons/shapes/media';
-import { SocialShapes } from '@clr/icons/shapes/social';
-import { TechnologyShapes } from '@clr/icons/shapes/technology';
-import { TextEditShapes } from '@clr/icons/shapes/text-edit';
-import { TravelShapes } from '@clr/icons/shapes/travel';
+import { ClrChartSet } from '@clr/icons/shapes/chart';
```

Comment:
Should we be importing from '@clr/icons' for the sets?

Reply:
Yup, I made the change.